commandline/library/micronucleus_lib.c

```diff
@@ -97,6 +99,11 @@ micronucleus* micronucleus_connect(int fast_mode) {
 
           assert(res >= 6);
```

Comment:
This assert should also be adjusted if the descriptor length is increased.

Reply:
It must be at 6 to be compatible with older bootloaders sending only 6 bytes.

Reply:
ah ok